packages/grid/_modules_/grid/hooks/features/keyboard/useKeyboard.ts

```diff
@@ -225,30 +228,24 @@ export const useKeyboard = (gridRootRef: React.RefObject<HTMLDivElement>, apiRef
       if (isMultipleKey(event.key)) {
         logger.debug('Multiple Select key pressed');
         onMultipleKeyChange(true);
-        return;
       }
 
-      if (!isCell(document.activeElement)) {
+      if (!isCellRoot(document.activeElement)) {
         return;
       }
 
```

Comment:
```suggestion
     if (!isTabKey(event.code)) {
        // WE prevent default behavior for all key shortcut except tab when the current active element is a cell
        event.preventDefault();
        event.stopPropagation();
      }
```

@oliviertassinari  btw if we return this `if` statement it will fix #797  🤔

Reply:
Agree, I think that we will need to introduce the prevent default back. I think that we should not include the stop propagation event as it's generally considered a footgun unless there a strong case for preventing another listener to handle the event.

Regarding where we should introduce it back, I haven't looked, but maybe inside each if statement that might have a global impact, like the select all, arrow, etc. In the very first version of Damien, we were too aggressive, we were preventing default everything.

Reply:
Ok, I've added the `event.preventDefault();` only on the navigation parts and the select all shortcut, not on the copy shortcut. I've tested it, this does solve #797 and no other regressions are present.

PS: I know that that this is again looking like a multipurpose PR but I can mark it as a bulk fix since all the issues are related to the same piece of code.

Reply:
It seems to work correctly with the latest commit: https://codesandbox.io/s/material-demo-forked-vfcto?file=/demo.js.
We can add one simple test case as a smoke a move forward, great 👍